experimental/LieAlgebras/src/LieAlgebras.jl

```diff
@@ -134,15 +146,21 @@ export LieAlgebraModule, LieAlgebraModuleElem
 export LieAlgebraModuleHom
 export LieSubalgebra
 export LinearLieAlgebra, LinearLieAlgebraElem
+export RootSystem
+export SimpleLieAlgebra, SimpleLieAlgebraElem
 
 export abelian_lie_algebra
 export abstract_module
+export adjoint_matrix
 export base_lie_algebra
 export base_module
 export base_modules
 export bracket
 export coerce_to_lie_algebra_elem
 export derived_algebra
+export cartan_matrix
+export chevalley_basis
+export dynkin_diagram
```

Comment:
```suggestion
export cartan_matrix
export chevalley_basis
export coerce_to_lie_algebra_elem
export derived_algebra
export dynkin_diagram
```
keep alphabetical order